environments.json

```diff
@@ -2008,7 +2008,7 @@
     "family": "Duktape",
     "short": "DUK 2.2",
     "platformtype": "engine",
-    "unstable": true
+    "obsolete": false
```

Comment:
You could add the release date here.  Actually, I guessed the "retire date" (end of support) for all the previous versions, you could check them and add the retire date for 2.1 too. ;)

Reply:
Hmm, 2.1 will still get at least one round of maintenance fixes so I guess it's not retired yet?

Reply:
Sure.  But the release data from 2.2 is still missing. :)

Reply:
No disagreement there, I'll add it.

Reply:
Actually, your comment on 2.1 still having one round yet makes me doubt of the retire dates I guessed for all other versions.  Can you update or remove them?